docs/coaches_training.deck.md

```diff
@@ -331,13 +324,21 @@ DISCUSS!
 * Co-coaches can help people who get lost.
 * If someone falls behind, the co-coach can take them aside to do some 1-on-1.
 
-# Discussion: Comprehension
+# The Curriculum
+* Most people will go through the curriculum on our web page
 
-#### How can you tell if they understand the words you're saying?
-#### What are good questions to ask to check comprehension?
-#### What did your favorite coaches do to gauge understanding?
+http://clojurebridge-berlin.org/curriculum
 
-DISCUSS!
+### It's a slideshow that goes through the concepts of:
+* values
+* data structures
+* functions
+* control logic
+
+## Please take a look :)
+
+# Some people are trialing a new Curriculum
```

Comment:
nit: lowercase c

Reply:
"slideshow that goes through the concepts....and then people play with Quil"?